stories/1-Dropdown.stories.tsx

```diff
@@ -1,35 +1,78 @@
-import React from "react";
+import React, {Fragment} from "react";
 import {storiesOf} from "@storybook/react";
 
 import Dropdown from "../src/dropdown/Dropdown";
 import StateProvider from "./utils/StateProvider";
 import FormField from "../src/form/field/FormField";
+import StoryFragment from "./utils/StoryFragment";
 
-storiesOf("Dropdown", module).add("Dropdown States", () => {
-  const initialState = {
-    options: [
-      {
-        id: "turkish",
-        title: "Turkish"
-      },
-      {
-        id: "english",
-        title: "English"
-      },
-      {
-        id: "spanish",
-        title: "Spanish"
-      },
-      {
-        id: "french",
-        title: "French - Disabled",
-        isDisabled: true
+const initialState = {
```

Comment:
Let's define the types for these variables. Also lets try to move them to a separate module, e.g. `dropdownStoryConstants`